src/core_atmosphere/diagnostics/cloud_diagnostics.F

```diff
@@ -0,0 +1,209 @@
+! Copyright (c) 2016,  Los Alamos National Security, LLC (LANS)
+! and the University Corporation for Atmospheric Research (UCAR).
+!
+! Unless noted otherwise source code is licensed under the BSD license.
+! Additional copyright and license information can be found in the LICENSE file
+! distributed with this code, or at http://mpas-dev.github.com/license.html
```

Comment:
We haven't always done a good job at keeping the license and copyright info up to date, but since we're creating a new code module, we might as well update a few items here:
1) Change 2016 to 2022, and probably just make UCAR the copyright holder
2) I think the correct URL for the license document (which is definitely out of date) is https://mpas-dev.github.io/license.html .

Reply:
Can do!

Reply:
I think the updated year should be 2022 rather than 2020.

Reply:
Whoops, these two things should be fixed with the force push